charts/consul/values.yaml

```diff
@@ -654,11 +654,59 @@ global:
     # self-managed cluster.
     enabled: false
     
-    # The name of the Kubernetes secret that holds the HCP cloud configuration.
-    # It contains the HCP service principal client_id and client_secret as well
-    # as the HCP resource_id.
-    # @type: string
-    secretName: null
+    # The name of the Kubernetes secret that holds the HCP resource id.
```

Comment:
Is there a way to indicate which of these are required and which are optional?

Reply:
done.  thank you for the suggestion!